cloudinary_storage/storage.py

```diff
@@ -101,7 +101,7 @@ def _prepend_prefix(self, name):
         prefix = self._normalize_path(prefix)
         if not name.startswith(prefix):
             name = prefix + name
-        return name
+        return name.replace('\\', '/')
```

Comment:
I think line 104 could be extracted as separate method for readability, such as:
```python
def _convert_to_unix_path(path):
    return path.replace('\\', '/')
```

Then it could be used at the beginning of `_save`, like:
```python
def _save(self, name, content):
    name = self._convert_to_unix_path(name)
    name = self._prepend_prefix(name)
    content = UploadedFile(content, name)
    response = self._upload(name, content)
    return response['public_id']
```

Apart from readability, it would be a little safer as replacing backslashes with slashes at the end of `_prepend_prefix` is dangerous as for example `_normalize_path` would append unnecessary `/` to windows style path due to `if path != '' and not path.endswith('/')` condition.